elastiknn-api4s/src/test/scala/com/klibisz/elastiknn/api/XContentCodecSuite.scala

```diff
@@ -110,13 +110,16 @@ class XContentCodecSuite extends AnyFreeSpec with Matchers {
           ("L2", Similarity.L2),
           ("cosine", Similarity.Cosine),
           ("Cosine", Similarity.Cosine),
-          ("COSINE", Similarity.Cosine)
+          ("COSINE", Similarity.Cosine),
```

Comment:
General comment for this file: Let's also add a block of tests below similar to this: https://github.com/alexklibisz/elastiknn/blob/39fa6109b3f03b5d03dbba404ead70b861fddceb/elastiknn-api4s/src/test/scala/com/klibisz/elastiknn/api/XContentCodecSuite.scala#L371-L406